src/sys/unix.rs

```diff
@@ -204,6 +204,9 @@ const MAX_BUF_LEN: usize = ssize_t::MAX as usize;
 #[cfg(target_vendor = "apple")]
 const MAX_BUF_LEN: usize = c_int::MAX as usize - 1;
 
+#[cfg(all(feature = "all", any(target_os = "freebsd", target_os = "linux")))]
+const TCP_CA_NAME_MAX: usize = 16;
```

Comment:
Let's add this constant to libc.

Reply:
Hey, `TCP_CA_NAME_MAX` isn't defined in user space include files, this is a kernel-side definition. So what should I do?

Reply:
Can you add a comment saying so, I prefer to use `libc`'s constant, but if that's not possible it's ok to define it ourselves.

Reply:
Ok I will add the comment.